software-engineer-in-test-US.md

```diff
@@ -0,0 +1,83 @@
+
+- Take ownership and work autonomously on any development task within an application or service, delivering code on time and with a quality mindset.
+
+### You may be a good fit if you have
+- Significant experience building test automation solutions, automated testing frameworks, monitoring systems and improving test effectiveness.
+Experience working with public cloud infrastructure and services.
+
+- Experience in all or a subset of: 
+    * Automated end-to-end testing of web applications.
+    * Automated end-to-end testing of native mobile applications Automated testing and verification of HTTP APIs.
+    * Automated performance, load and soak testing of user interfaces or APIs. 
+    * Automated infrastructure testing or chaos testing.
+    * Automated security and vulnerability testing.
+    * Ability to determine the right open-source/tool to address tooling requirements.
+    * Ability to write and understand unit tests. 
+    * Know or are willing to learn "go" programming language. 
+    * Basic knowledge of mobile app test automation.
+    * Practical experience building quality for microservices.
+    * Practical experience building quality for distributed systems containing several microservices.
+    * Understanding of how best practises with Quality Engineering.
+    * Understanding of technologies like traceability, cloud computing and Kubernetes.<BR>
+
+- Experience working with production-grade JavaScript.
+- Significant experience in communicating and working with software teams, and working towards building inclusive, fun environments.
+- Experience with the US Banking domain is preferred but not a mandatory condition.
+- A passion for learning and growth, picking up new skills and tools.
+- Solid knowledge on front-end technologies.
+- A track record of developing system maintainability and ease of operation.
+- Deep understanding of the development process, design, development, testing and deployment.
+- Flexible to work with teams in multiple time zones.
+
+### The interview process
+The interview process will focus on three areas: your experience building products and taking on technical challenges; your technical skills; and how you'll fit in culturally with the rest of the team.<BR>
+We want to take your time seriously, and use as little of it as possible to make our decision. We will also commit to giving you constructive, meaningful feedback whether we decide to move forward together or not.<BR>
+All interviews will take place remotely, via Zoom.<BR>
+Your interview will be broken down into three stages, scheduled separately and with feedback after each: 
+- A 30-minute interview with your team's Engineering Lead, where we'll tell you about the role, understand your experience, and professional background and understand common interests. At the end of this meeting, we will provide a coding challenge.  The candidate shall provide a solution and share it via GitHub, within 1-2 days.
```

Comment:
Do I get to be in any of the interviews? 😁

Reply:
yes. ofcourse.